rgw/v2/tests/s3cmd/configs/test_rgw_log_details.yaml

```diff
@@ -0,0 +1,9 @@
+# script: test_rgw_ops_log.py
+# Bugs: 2188557
+# Polarian: CEPH-83575470
+config:
+  user_count: 1
+  bucket_count: 2
+  objects_count: 100
+  test_ops:
+    rgw_log_detail: true
```

Comment:
@anrao19 , are we intentionally not using `rgw_ops_log: true` here
I dont see any of s3 ops like upload object etc.. in the log
http://magna002.ceph.redhat.com/cephci-jenkins/Anuchaithra/test_rgw_log_details.console.log

Reply:
test_ops is do just for the if condition , not set anything.
Here we are just getting rgw log details, if log_to_file not set we will set then we will get the details